ts/packages/anchor/src/utils/events.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  CompiledInnerInstruction,
+  VersionedTransactionResponse,
+} from "@solana/web3.js";
+import { BorshEventCoder } from "src/coder";
+import { Idl } from "src/idl";
+import { IdlEvents } from "src/program";
+
+// https://github.com/coral-xyz/anchor/blob/v0.29.0/tests/events/tests/events.ts#L61-L62
+export function parseCpiEvents(
```

Comment:
Why not include this as a method of `EventCoder`/`BorshEventCoder`?

Reply:
Changed :+1: